packages/now-cli/src/commands/env/add.ts

```diff
@@ -146,3 +148,7 @@ export default async function add(
 
   return 0;
 }
+
+function isSystemEnvVariable(envName: string) {
+  return envName.startsWith('VERCEL_') || envName.startsWith('NOW_');
```

Comment:
```suggestion
  return envName.startsWith('VERCEL_');
```

Because we don't support adding `NOW_*` system env variable anymore.